### PR TITLE
add option for newcommand on hover preview

### PR DESCRIPTION
**Pull request by tamuratak**
Added an option that a TeX file in which newcommands defined can be used for hover preview. Requested in #1071.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1128,6 +1128,16 @@
...
         },
+        "latex-workshop.hover.preview.newcommand.parseTeXFile.enabled": {
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Enable newcommands defined in the current TeX file to be included in Hover Preview."
+        },
+        "latex-workshop.hover.preview.newcommand.newcommandFile": {
+          "type": "string",
+          "default": "",
+          "markdownDescription": "Set the path of a file in which newcommands of LaTeX defined for Hover Preview. If the path is relative, it is joined with the root dir."
```
Comment (jlelong) [inline package.json]: I guess there is a typo in the description. Can you fix this?

**Comment by jlelong** [comment]
Could you also make a PR to describe this feature in the [wiki](https://github.com/jlelong/LaTeX-Workshop-wiki/blob/master/Hover.md)?

File: package.json (hunk 1 above)
Comment (tamuratak) [reply package.json]: I cannot find. Please tell me.

Comment (jlelong) [reply package.json]: What about: "Set the path of a file containing newcommands to be used in Hover Preview. If the path is relative, it is joined with the root dir."

Comment (tamuratak) [reply package.json]: Thanks.

**Comment by tamuratak** [comment]
@jlelong Sure. I will make a PR later.

**Pull request merged**